exercises/sum.cpp

```diff
@@ -6,3 +6,25 @@
   Insert the second number: 2
   Sum: 3
 */
+
+#include <iostream>
+
+using namespace std;
+
+double sum(double x, double y) {
+  return x + y;
+}
+
+int main() {
+  double x, y;
+
+  cout << "Insert the first number: ";
+  cin >> x;
+
+  cout << "Insert the second number: ";
+  cin >> y;
+
+  cout << "Sum: " << sum(x, y) << endl;
+
+  return 0;
```

Comment:
potresti sostituire 0 con EXIT_SUCCESS in modo da migliorare la leggibilità del codice
```c++
#define EXIT_SUCCESS 0
...
return EXIT_SUCCESS;
```